README.md

```diff
@@ -1,42 +1,138 @@
-# Scotch
+# EasyVCR
 
-### Disclaimer
-This is a fork of the original [Scotch by Martin Leech](https://github.com/mleech/scotch), updated to address outstanding functionality issues, as well as introduce a handful of new features. This version of Scotch is a C# rewrite of the original F# version.
+#### Disclaimer
```

Comment:
Let's move this to the bottom of the README, more of an afterthought than the most exciting/important introduction to the project.